openhgnn/trainerflow/SeHGNN_trainer.py

```diff
@@ -0,0 +1,514 @@
+import os
+import gc
+import time
+import uuid
+import argparse
+import datetime
+import numpy as np
+
+import torch
+import torch.nn.functional as F
+
+import os
+import sys
+import gc
+import random
+import sparse_tools
+import dgl
+import dgl.function as fn
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch_sparse import SparseTensor
+from torch_sparse import remove_diag, set_diag
+
+import numpy as np
+import scipy.sparse as sp
+from sklearn.metrics import f1_score
+from tqdm import tqdm
+from ogb.nodeproppred import DglNodePropPredDataset, Evaluator
+from openhgnn.models import build_model
+from . import BaseFlow, register_flow
+from ..tasks import build_task
+import functools
+from contextlib import closing
+import multiprocessing as mp
+from multiprocessing import Pool
+from tqdm import tqdm
+from ..tasks import NodeClassification
+
+@register_flow("SeHGNN_trainer")
+class SeHGNNtrainer(BaseFlow):
```

Comment:
This class is comfusing. I recommend you to rewrite this refer to the `openhgnn.trainer_flow.node_classification.py`.